google-cloud-spanner/test/google/cloud/spanner/batch_snapshot/execute_partition_test.rb

```diff
@@ -260,7 +260,7 @@
       columns: ["id", "name", "active", "age", "score", "updated_at", "birthday", "avatar", "project_ids"],
       key_set: Google::Cloud::Spanner::V1::KeySet.new(all: true),
       transaction: tx_selector, index: "", limit: nil, resume_token: nil, partition_token: partition_token,
-      request_options: nil
+      request_options: nil, data_boost_enabled: false
```

Comment:
If we have defined `data_boost_enabled ` with default value as `false` in function signature, then can we skip it here?

Reply:
Yup it is defined with default value as false, the change here is for the mock method to expect it with the default value.